src/classes.scss

```diff
@@ -25,6 +25,7 @@
 @import './modal/classes';
 @import './non-ideal-state/classes';
 @import './notification-marker/classes';
+@import './popover/classes';
```

Comment:
Not sure whether it should be `popover` folder or we might need to have some utils or react folder for these kind of stuff.

Reply:
Sure we wanna add all of that in our main bundle? Maybe lets just keep this as separate css file?

Reply:
That might require changing our compile script

Reply:
Idk, if imports in react won't work (e.g. Next.js) then they may decide to import `all.css` and then it will not have it these new rules.

Reply:
Just not include that in `classes.scss` and explicitly compile popover sass in script

Reply:
Isn't that too much for  a few lines?
![image](https://user-images.githubusercontent.com/36186912/152386090-4202c5cc-abcb-4561-8c03-3e203e0b34da.png)

Reply:
ahh...
well w/e then..
as long as you are sure you are not reverting someting which might influence the result...